Add Bored Ape Yacht Club to ERC-721 developer docs (#4605)
ethereum/ethereum-org-website: src/content/developers/docs/standards/tokens/erc-721/index.md

## Popular NFTs {#popular-nfts}

- [Etherscan NFT Tracker](https://etherscan.io/tokens-nft) list the top NFT on Ethereum by transfers volume.

> - [CryptoPunks](https://larvalabs.com/cryptopunks) is a collection of 10,000 unique characters, algorithmically generated.

Reviewer: While I agree CryptoPunks are popular NFTs, they don't actually comply w/ ERC-721, so I wonder if we should add them to this page. May cause confusion.

Reviewer 2: I did not know this!

Reviewer 2: After researching CryptoPunks a little more I agree with Sam that we shouldn't include them on this page. My thoughts are that the primary focus of the ERC-721 developer doc is to explain the token standard, explain how to implement it, and give examples of its implementation. 

I'm good with removing CryptoPunks and adding Bored Apes 🤘🦍

```suggestion
```

Reviewer 2: After researching CryptoPunks a little more I agree with Sam that we shouldn't include them on this page. My thoughts are that the primary focus of the ERC-721 developer doc is to explain the token standard, explain how to implement it, and give examples of its implementation. 

I'm good with removing CryptoPunks and adding Bored Apes 🤘🦍

```suggestion
```

Author: Hi Sam, Thanks for the fix!
In fact they are a custom smart contract released before the ERC-721 standard,
and as you say, it might confuse some users💪

Reviewer 2: ```suggestion
```

  censorship-resistant websites.
- [Gods Unchained Cards](https://godsunchained.com/) is a TCG on the Ethereum blockchain that uses NFT's to bring real ownership
  to in-game assets.

> - [Bored Ape Yacht Club](https://boredapeyachtclub.com) is a collection of 10,000 unique NFTs, which, as well as being a provably-rare piece of art, acts as a membership token to the club, providing member perks and benefits that increase over time.

Reviewer 2: ```suggestion
- [Bored Ape Yacht Club](https://boredapeyachtclub.com) is a collection of 10,000 unique NFTs, which, as well as being a provably-rare piece of art, acts as a membership token to the club, providing member perks and benefits that increase over time as a result of community efforts.
```

  censorship-resistant websites.
- [Gods Unchained Cards](https://godsunchained.com/) is a TCG on the Ethereum blockchain that uses NFT's to bring real ownership
  to in-game assets.

> - [Bored Ape Yacht Club](https://boredapeyachtclub.com) is a collection of 10,000 unique Bored Apes which gives you access to
>   member-only benefits and all areas of the project (future areas and perks can be unlocked by the community through roadmap activation).

Reviewer 2: I don't think the wording here is great. What about something like this?

```suggestion
- [Bored Ape Yacht Club](https://boredapeyachtclub.com) is a collection of 10,000 unique NFTs, which, as well as being a provably-rare piece of art, acts as a membership token to the club, providing member perks and benefits that increase over time.
```

Author: Hi Joshua,
I think this is much better and fine, Thanks.
But I ask you to add this final piece to give importance to the "work" of the community🙂

**" thanks to the community itself. "**

Reviewer 2: Done!
